### ISSUE TITLE
How to find keypaths in jetpack compose

### ISSUE THREAD

**Issue by Debdutta-Panda** [airbnb/lottie-android#2080]
I could not find any way or documentation to find all the keypaths in a animation. I got this XML approach but failed to do so in jetpack compose. So, please help to find all the keypaths in jetpack compose.

**Comment by gpeal** [comment]
Your best bet is probably to use an `AndroidView` composable with an actual `LottieAnimationView` and then switch back once you know the KeyPath you need or just open up the animation in a sample project. As of now, I'm not planning on porting that particular API.

**Comment by Debdutta-Panda** [comment]
Yes, have already used the technique. But I would like have this in compose way. The lottie is really very beautiful library for animation so it should be complete in itself. Its just my wish. Thanks.

**Comment by blundell** [comment]
I have a compose only project.

Requiring us to use `AndroidView` with `LottieAnimationView` means I have to add `androidx.appcompat:appcompat:_` as a gradle dependency, so thats not an ideal or quick solution.

**Comment by gpeal** [comment]
> I have a compose only project.
> 
> Requiring us to use `AndroidView` with `LottieAnimationView` to gain access to `resolveKeyPath()` means I have to add `androidx.appcompat:appcompat:_` as a gradle dependency, so thats not an ideal or quick solution.

You already have it as a transitive dependency

https://github.com/airbnb/lottie-android/blob/master/lottie/build.gradle#L45

**Comment by blundell** [comment]
Doesn't appear like that works. 

My guess would be because its an aar library, and you'd need to make a 'fat jar' if you wanted to include androidx (which I also think you might not want to?)

Made a minimal reproducer here: https://github.com/blundell/LottieErrorRepro

<img width="1354" alt="image" src="https://github.com/user-attachments/assets/421082ee-a732-4281-8780-9702dd252815" />

**Comment by gpeal** [comment]
@blundell Is there a reason you can't add appcompat?

**Comment by blundell** [comment]
In a bigger project it would probably always be used. 

I am just saying that the Android Studio new project template doesn't have it, and so just adding the two lottie dependencies needed, you get this error.
